app/src/main/java/com/parasoft/demoapp/component/OrderDialog.java

```diff
@@ -137,4 +168,107 @@ public void onFailure(@NonNull Call<ResultResponse<OrderResponse>> call, @NonNul
                     }
                 });
     }
+
+    public void setOrderLayout() {
+        if (orderInfo.getComments() == null || orderInfo.getComments().length() == 0) {
+            comments.setVisibility(View.GONE);
+        } else {
+            comments.setVisibility(View.VISIBLE);
+            commentsDetail.setText(orderInfo.getComments());
+            if(commentsDetail.getLineCount() == 1) {
+                commentsDetail.setGravity(Gravity.END);
+            } else {
+                commentsDetail.setGravity(Gravity.START);
+            }
+        }
+        SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss", Locale.getDefault());
+        String date = sdf.format(orderInfo.getSubmissionDate());
+        orderTimeYear.setText(date.substring(0, 10));
+        orderTimeHour.setText(date.substring(11));
+        orderStatus.setText(getStatus(orderInfo.getStatus()));
+        purchaserName.setText(orderInfo.getRequestedBy());
+        location.setText(getRegion(orderInfo.getRegion()));
+        receiverName.setText(orderInfo.getReceiverId());
+        gpsCoordinates.setText(orderInfo.getLocation());
+        map.setImageDrawable(getMap(orderInfo.getOrderImage()));
+    }
+
+    public String getRegion(String region) {
+        switch (region) {
+            case "LOCATION_1" :
```

Comment:
LOCATION_x just for outdoor industry, there are other regions for other two industries.